languages/french.json

```diff
@@ -9,162 +9,90 @@
-		"ExtCannotLoad": "Impossible de charger cette extension. Merci de le signaler.",
 		"All": "Tout",
-		"NoReadme": "Aucun fichier \"readme\" trouvé.",
 		"Stars": "Stars",
-		"MarketError1": "Impossible de lire la liste des plugins.",
-		"MarketError2": "Impossible de lire plus de fois la liste des plugins.",
-		"MarketError3": "Impossible de lire le fichier \"package.json\" de la source du plugin.",
-		"SetupError1": "Impossible d'installer les thèmes par défaut, vous pourrez les installer plus tard via le marché.",
-		"Welcome.AdditionalSettings": "Paramètres additionnels",
-		"OpenInExplorer": "Ouvrir dans l'explorateur",
-		"Cache": "Cache",
-		"Clear": "Vider le cache",
-		"MiniMap": "MiniMap",
-		"MiniMapDisabled": "Redémarrez Graviton pour prendre en compte les nouveaux paramètres",
-		"CopyPath": "Copier le chemin",
-		"NoUpdateFound": "Aucune mise-à-jour disponible",
-		"ExtNotListed": " n'existe pas dans le marché.",
-		"OpenAsGlobal": "Utiliser comme dossier racine.",
-		"DetectedUpdateMessage": "Voulez-vous mettre à jour votre version",
-		"Readme": "Readme",
-		"Permissions": "Permissions",
-		"PermissionCustomIcons": "Personnalizer les icônes des fichiers",
-		"PermissionCustomStyling": "Personnaliser le style de l'interface",
-		"PermissionCustomColors": "Personnaliser les couleurs de l'interface",
-		"PermissionExecuteJavaScript": "Exécuter du code JavaScript",
+		"NewFolder": "Nouveau dossier",
+		"NewFile": "Nouveau fichier",
+		"Report": "Signaler",
+		"Clear": "Vider",
 		"Edit": "Editer",
 		"Undo": "Annuler",
 		"Redo": "Refaire",
-		"SystemAccentColor.Error1": "Votre système n'est pas compatible avec cette fonctionnalité.",
-		"SystemAccentColor.Error2": "L'utilisation des couleurs d'accent du système pourrait ne pas fonctionner correctement avec le thème actuel.",
-		"TelegramChannel": "Cannal Telegram",
-		"Settings-Advanced-Developer-Tools-text2": "Ouvrir le dossier de configuration",
-		"ExplorerPosition": "Position de l'explorateur de dossiers",
-		"Screenshots": "Captures d'écran",
-		"NoRecentProjects": "Aucun projet récent.",
-		"Miscellaneous": "Divers",
-		"Explorer": "Explorateur",
-		"Process.DownloadingSource": "Téléchargement du code source...",
-		"Process.CheckingDependencies": "Vérification des dépendances...",
-		"Process.InstallingDependencies": "Installation des dépendances...",
-		"Process.Completed": "Terminé !",
-		"Hide":"Cacher",
-		"Show":"Montrer",
-		"NPMPanel":"Panneau NPM",
-		"Select":"Sélectionner",
-		"By":"Par",
-		"Suggestions":"Suggestions"
+		"NoRecentProjects": "Pas de projets récent",
+		"Hide": "Cacher",
+		"Show": "Montrer",
+		"Select": "Selectionner",
+		"By": "Par",
+		"Suggestions": "Suggestions",
+		"OpenProjects": "Projets ouverts",
+		"OpenWorkspaces": "Espaces de travail ouverts",
+		"OpenWorkspaceFromFile": "Ouvrir un espace de travail à partir d'un fichier",
+		"AddFolderToWorkspace": "Ajouter les dossier à l'espace de travail",

```

Comment:
I'm not sure about the context, but maybe `un dossier` instead of `des dossier` ?

Reply:
I'm not sure but it's patched. If we need to, we can create an other PR or ask to Marc to change it.

Reply:
Yes ^^